notebooks/elasticsearch-spark-recommender.ipynb

```diff
@@ -990,6 +1005,27 @@
     "        if i % 5 == 0:\n",
     "            sim_html += \"</tr><tr>\"\n",
     "    sim_html += \"</tr></table>\"\n",
+    "    display(HTML(sim_html))\n",
+    "    \n",
+    "def display_similar_users(the_id, q=\"*\", num=10, index=\"demo\", dt=\"users\"):\n",
+    "    \"\"\"\n",
+    "    Display similar users and similarity scores, in a table\n",
+    "    \"\"\"\n",
+    "    user, recs = get_similar_users(the_id, q, num, index, dt)\n",
```

Comment:
Hence per my comment above, this call could just be `user, recs = get_similar(the_id, q, num, index, dt="users")`